### PR TITLE
update willReadFrequently in getContext and getContextAttributes

### PR DESCRIPTION
**Pull request by yiyix**
willReadFrequently is one of the canvas attribute can be used to create canvas context. I updated its description in `getContext` and `getContextAttributes`.

### CODE REVIEW

**Comment by jpmedley** [comment]
Thanks for submitting. What is getContext() a member of.

**Comment by yiyix** [comment]
> Thanks for submitting. What is getContext() a member of.

It's member of HTMLCanvasElement, link: https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext

File: content/en-US/api/getContext.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+willReadFrequently: Boolean that indicates if the `CanvasRenderingContext2D` will be read frequently.
+On most devices, user agent needs to decide whether to store the canvas's bitmap on GPU or on the
```
Comment (jpmedley) [inline content/en-US/api/getContext.md]:
```suggestion
On most devices, user agent needs to decide whether to store the canvas's bitmap on the GPU or on the
```

File: content/en-US/api/getContext.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+CPU. Rendering operations are more performant for accelerated canvases (on the GPU) except
+doing readback with `getImageData()`, which is better on software canvas (on the CPU).
```
Comment (jpmedley) [inline content/en-US/api/getContext.md]:
```suggestion
when doing readback with `getImageData()`, which is better on software canvas (on the CPU). 
```

File: content/en-US/api/getContextAttributes.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## Return value
```
Comment (jpmedley) [inline content/en-US/api/getContextAttributes.md]: Can you create a whole page for this method. Run `npm run create -- -i getContextAttributes` and select "Property" from the list.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: content/en-US/api/getContextAttributes.md (hunk 3 above)
Comment (yiyix) [reply content/en-US/api/getContextAttributes.md]: I am not sure if there is enough to create a whole page for this one. I just want to add one more return value, one paragraph. I put alpha and desynchronized to show the location for this paragraph. 
I also searched for alpha and desynchronized, both of them don't have they own page as well.

Comment (jpmedley) [reply content/en-US/api/getContextAttributes.md]: I understand, but MDN is not a blog. It's more like a dictionary. It needs an entry for every interface, constructor, event, event handler, method, and property.

Comment (yiyix) [reply content/en-US/api/getContextAttributes.md]: I see, I added a new file for this property. Could you review again? Thank you.

File: content/en-US/api/willReadFrequently.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+recipe: api-interface
+title: 'ContextAttributes.willReadFrequently'
+mdn_url: /en-US/docs/Web/API/ContextAttributes/willReadFrequently
+specifications: https://html.spec.whatwg.org/multipage/canvas.html#dom-canvasrenderingcontext2dsettings-willreadfrequently
+browser_compatibility: api.ContextAttributes.willReadFrequently
+---
+
+## Description
+
+The `willReadFrequently` is property of the `ContextAttributes` interface returns a boolean
```
Comment (jpmedley) [inline content/en-US/api/willReadFrequently.md]:
```suggestion
The `willReadFrequently` property of the `ContextAttributes` interface returns a boolean
```

File: content/en-US/api/willReadFrequently.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+value that indicates if the `CanvasRenderingContext2D` will be read frequently and user agent
```
Comment (jpmedley) [inline content/en-US/api/willReadFrequently.md]:
```suggestion
that indicates if the `CanvasRenderingContext2D` will be read frequently and the user agent
```

File: content/en-US/api/willReadFrequently.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+needs to optimize readback with `getImageData()`. 
+
+On most devices, user agent needs to decide whether to store the canvas's bitmap on the GPU or
```
Comment (jpmedley) [inline content/en-US/api/willReadFrequently.md]:
```suggestion
On most devices, the user agent needs to decide whether to store the canvas's bitmap on the GPU or
```

File: content/en-US/api/willReadFrequently.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+
+## Syntax
+
+`HTMLCanvasElement.getContext('2d', {willReadFrequently : true})`
+`CanvasRenderingContext2D.getContextAttributes().willReadFrequently`
+
+### Value
+
+A boolean value which indicates whether the CanvasRenderingContext2D is marked for readback
```
Comment (jpmedley) [inline content/en-US/api/willReadFrequently.md]:
```suggestion
A boolean value which indicates whether the `CanvasRenderingContext2D` object is marked for readback
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
This looks pretty good. Thank you for the work. There are a few typos.

File: content/en-US/api/willReadFrequently.md (hunk 4 above)
Comment (yiyix) [reply content/en-US/api/willReadFrequently.md]: done

**Pull request merged**